github_connector/__manifest__.py

```diff
@@ -6,7 +6,7 @@
 {
     'name': 'Github Connector',
     'summary': 'Synchronize information from Github repositories',
-    'version': '12.0.1.1.0',
+    'version': '12.0.1.1.1',
```

Comment:
This is not wrong per se, but the recommendation is not to do this. When we instruct the bot to merge, it will handle the version bump. This helps avoiding merge conflicts if you need to merge several PRs.